ReactiveXComponentTest/IntegrationTests/RabbitMqCommunicationTest.cs

```diff
@@ -33,23 +36,136 @@ public void Setup()
             };
             _exchangeName = "3";
```

Comment:
You should generate exchangeName and routing key in order to work with test parallel runs